src/tests/unit/tests/DetailsView/components/switcher.test.tsx

```diff
@@ -36,21 +36,20 @@ describe('Switcher', () => {
         expect(renderer.debug()).toMatchSnapshot();
     });
 
-    test('onOptionClick', () => {
+    test('onOptionChange', () => {
         actionCreatorMock
-            .setup(creator => creator.sendPivotItemClicked(DetailsViewPivotType[DetailsViewPivotType.fastPass]))
+            .setup(creator => creator.sendPivotItemClicked(DetailsViewPivotType[DetailsViewPivotType.assessment]))
```

Comment:
do you need to swap the states like this??? It feels like adding a change that doesn't actually affect what you're testing.

Reply:
Just want to make the test scenario more clear: initial state is fastpass and user changes it to assessment.

Reply:
Why is the reverse not good enough?

Reply:
The initial state was set as fastPass (on line 19), so it feels more straightforward to me that we are testing state change from fastPass to assessment.